src/js/components/HeaderBar.tsx

```diff
@@ -18,6 +21,19 @@ export default class HeaderBar extends React.Component {
           </Link>
         </div>
         <div className="header-bar-right-align-wrapper">
+          {refreshRate > 2 ? (
+            <span>
+              <Tooltip
+                id="refreshRate"
+                trigger={`Refresh Rate: ${refreshRate} sec`}
+                maxWidth={150}
+              >
+                Data will only be requested every {refreshRate} seconds per
+                endpoint
```

Comment:
let's remove the "per endpoint" here as users most likely won't care and it also might not be true if we throttle repainting on some pages.